dotnetv3/KMS/README.md

```diff
@@ -1,59 +1,47 @@
-# AWS Key Management Service (AWS KMS) .NET examples
-Purpose
+# AWS KMS code examples for .NET
 
-The code examples in this directory demonstrate how to work with AWS KMS using
-the AWS SDK for .NET v3.5 or later.
+## Overview
 
-AWS KMS presents a single control point to manage keys and define policies
-consistently across integrated AWS services and your own applications. AWS KMS
-is integrated with AWS CloudTrail to provide you with logs of all key usage to
-help meet your regulatory and compliance needs.
+The examples in this section show how to use the AWS SDK for .NET with AWS Key Management Service (AWS KMS) to create and manage keys, grants, and policies.
 
-## Code examples
+AWS KMS lets you create, manage, and control cryptographic keys across your applications and more than 100 AWS services.
```

Comment:
AWS KMS is an encryption and key management service scaled for the cloud. AWS KMS keys and functionality are used by other AWS services, and you can use them to protect data in your own applications that use AWS.

Reply:
Fixed.